### PR TITLE
Use the configuration file in DBus modules

### PR DESCRIPTION
**Pull request by poncovka**
The Anaconda DBus launcher is able to set up and clean up everything
related to the DBus modules. It will create a temporary configuration
file, start a DBus session, set up the environment, start the boss
and kickstart modules and wait for the modules to start. It raises
TimeoutError if the modules don't start on time.

The list of enabled kickstart modules is provided by the configuration file
now.

### CODE REVIEW

File: pyanaconda/dbus/launcher.py [hunk 1]
```diff
@@ -130,3 +148,53 @@ def stop(self):
...
+        file_name = ANACONDA_BUS_ADDR_FILE
+        run_dir = os.path.dirname(file_name)
+
+        if not os.path.exists(run_dir):
+            os.mkdir(run_dir)
+
+        with open(file_name, 'wt') as f:
+            f.write(self._bus_address)
+
+    def _remove_bus_address(self):
```
Comment (jkonecny12) [inline pyanaconda/dbus/launcher.py]: I would rename this to `_remove_bus_address_file` to be more explicit.

File: pyanaconda/modules/common/__init__.py [hunk 2]
```diff
@@ -31,3 +31,10 @@ def init():
     import locale
     from pyanaconda.core.constants import DEFAULT_LANG
     locale.setlocale(locale.LC_ALL, DEFAULT_LANG)
+
+    from pyanaconda.core.configuration.anaconda import conf
+    # The configuration is validated on the first import.
```
Comment (jkonecny12) [inline pyanaconda/modules/common/__init__.py]: Please move this comment above the line.

File: pyanaconda/dbus/launcher.py (hunk 1 above)
Comment (poncovka) [reply pyanaconda/dbus/launcher.py]: Fixed.

File: pyanaconda/modules/common/__init__.py (hunk 2 above)
Comment (poncovka) [reply pyanaconda/modules/common/__init__.py]: Fixed.

**Comment by poncovka** [comment]
Added support for loading configuration files from /etc/anaconda/conf.d, so the Initial Setup can use it.

**Pull request merged**